### PR TITLE
Adding a bunch of missing redirects from the tutorial changes

### PR DESCRIPTION
**Pull request by tjvantoll**
I totally forgot about some of the older URL patterns that still live in Google land :(

This list is long but I think we need them all.

### CODE REVIEW

File: build/nginx.conf [hunk 1]
```diff
@@ -154,9 +154,33 @@ http {
 		rewrite ^/angular/tutorial/ng-chapter-5 $proto://$host_and_port/angular/start/introduction permanent;
 		rewrite ^/angular/tutorial/ng-chapter-6 $proto://$host_and_port/angular/start/introduction permanent;
 		rewrite ^/angular/tutorial/ng-chapter-7 $proto://$host_and_port/angular/start/introduction permanent;
+		rewrite ^/angular/tutorial/ng-chapter-7 $proto://$host_and_port/angular/start/introduction permanent;
```
Comment (etabakov) [inline build/nginx.conf]: For the sake of brevity, can we replace this group of redirects with a rule like this:

rewrite ^/(angular|start)/tutorial/ng-chapter-\d$ $proto://$host_and_port/angular/start/introduction permanent;

File: build/nginx.conf [hunk 2]
```diff
@@ -154,9 +154,33 @@ http {
...
+
+		rewrite ^/start/hello-world/ng-chapter-0 $proto://$host_and_port/angular/start/introduction permanent;
+		rewrite ^/start/hello-world/ng-chapter-1 $proto://$host_and_port/angular/start/introduction permanent;
+		rewrite ^/angular/start/hello-world/ng-chapter-0 $proto://$host_and_port/angular/start/introduction permanent;
+		rewrite ^/angular/start/hello-world/ng-chapter-1 $proto://$host_and_port/angular/start/introduction permanent;
 
 		# Redirect the JavaScript Groceries tutorial links
-		rewrite ^/tutorial/chapter-0 $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/getting-started $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/tutorial/chapter-0 $proto://$host_and_port/start/introduction permanent;
```
Comment (etabakov) [inline build/nginx.conf]: Again we can use something like:
rewrite ^/start/tutorial/chapter-\d$ $proto://$host_and_port/start/introduction permanent;

File: build/nginx.conf [hunk 3]
```diff
@@ -154,9 +154,33 @@ http {
...
+		rewrite ^/start/tutorial/chapter-1 $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/tutorial/chapter-2 $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/tutorial/chapter-3 $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/tutorial/chapter-4 $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/tutorial/chapter-5 $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/tutorial/chapter-6 $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/tutorial/chapter-7 $proto://$host_and_port/start/introduction permanent;
+		rewrite ^/start/tutorial/chapter-7 $proto://$host_and_port/start/introduction permanent;
```
Comment (etabakov) [inline build/nginx.conf]: I think this line is duplicated.

File: build/nginx.conf (hunk 1 above)
Comment (tjvantoll) [reply build/nginx.conf]: Yep. Testing this now.

**Comment by tjvantoll** [comment]
@etabakov Thanks! I just pushed a commit that should address all of your comments.

**Review by etabakov — APPROVED** [review]
Looks great 👍

**Comment by tjvantoll** [comment]
Merged with 68bcdf585088c8efdd0759b06b9204632d2ef8f0 and 6ccb3d399954fe8f9e6bf55b6b43849410a5b72b.

**Pull request closed without merging**